l10n_ec_account_edi/models/sri_key_type.py

```diff
@@ -222,3 +224,22 @@ def new_range():
         ctx.sign(signature)
         ctx.verify(signature)
         return etree.tostring(doc, encoding="UTF-8", pretty_print=True).decode()
+
+    def days_to_expire(self):
+        if self.expire_date:
+            return (self.expire_date - fields.Date.context_today(self)).days
+        return 0
+
+    def action_email_notification(self):
+        certificates = self.env["sri.key.type"].search([("state", "=", "valid")])
+        email_template = self.env.ref(
+            "l10n_ec_account_edi.email_template_notify", False
+        )
+        is_send = False
+
+        for cert in certificates:
+            if cert.days_to_expire() <= cert.days_for_notification:
```

Comment:
If the certificate is expired, I suspect that a notification is sent, but this is incorrect. The notification must not be sent. Please take a look.

Example OK
```
days_for_notification = 30
Today = 29/04/2024
Expire_date = 01/05/2024
days_to_expire = 3
3 < 30 = True So notification is OK
```

Example Wrong
```
days_for_notification = 30
Today = 29/04/2024
Expire_date = 20/04/2024
days_to_expire = -9
-9 < 30 =True So notification is wrong, because Certificate alread is expired
```


```suggestion
            if 0 < cert.days_to_expire() <= cert.days_for_notification:
```

Reply:
Yes, wrong